src/templates/React InstantSearch widget/src/lib/widget.tsx.hbs

```diff
@@ -0,0 +1,14 @@
+import { {{ pascalCaseName }}Component } from './component';
+import { connect{{ pascalCaseName }} } from './connector';
+
+import type { ElementType } from 'react';
+
+type WidgetParams = {
+  /**
+   * Placeholder text for input element.
+   */
+  placeholder?: string;
+};
+
+export const {{ pascalCaseName }}: ElementType<WidgetParams> =
```

Comment:
I'm not sure `ElementType` is the right one, but anything else like ReactElement, Component, etc didn't work.

By "didn't work", I mean TS gave me error messages like 

<img width="695" alt="CleanShot 2021-05-25 at 16 48 25@2x" src="https://user-images.githubusercontent.com/499898/119518767-1c64e980-bd79-11eb-9d3f-136a5ce7164a.png">

Reply:
as far as I can tell ElementType is correct

Reply:
it could also be ReactNode, did you try that?

Reply:
yes, ReactNode, ReactElement, all didn't work